log/handlers.ts

```diff
@@ -2,24 +2,48 @@ import { open, File, Writer } from "deno";
 import { getLevelByName } from "./levels.ts";
 import { LogRecord } from "./logger.ts";
 
+const DEFAULT_FORMATTER = "{levelName} {msg}";
```

Comment:
@ry should I set it to glog's format?

Reply:
No... it was just an idea and it seems people aren't so keen on it. Let's just move forward with this.